libs/fluid-elements/button-group/src/lib/button-group-item/button-group-item.ts

```diff
@@ -83,12 +83,10 @@ export class FluidButtonGroupItem extends LitElement {
       }
 
       :host([disabled]) .fluid-label {
-        // TODO: Should be replaced by an opacity token at some point.
         opacity: 0.5;
       }
 
       :host([disabled]) .fluid-button-group-item-svg {
-        // TODO: Should be replaced by an opacity token at some point.
```

Comment:
Why do you removed it?

Reply:
Very weird. It doesn't throw any errors anywhere but the opacity is not set when there's a comment above it. 🤔